rust/sdk-processor/src/parquet_processors/parquet_default_processor.rs

```diff
@@ -60,7 +62,25 @@ impl ProcessorTrait for ParquetDefaultProcessor {
             },
         }
 
-        // TODO: Starting version from config. 0 if not set.
+        // Determine the processing mode (backfill or regular)
+        let is_backfill = self.config.backfill_config.is_some();
+
+        // Query the starting version
+        let _starting_version = if is_backfill {
+            get_starting_version(&self.config, self.db_pool.clone()).await?;
```

Comment:
I don't remember if we decided on how backfill version tracking is going to work for Parquet? This function might not work well with Parquet because the version tracker "name" is different

Reply:
what's the version tracker name?

Reply:
the `processor_name` stored in DB

Reply:
oh yeah this needs an update. Added TODO: to follow up with version tracker change task